openreview/tools.py

```diff
@@ -1319,10 +1319,17 @@ def get_profile_info(profile, n_years=3):
     ## Relations section
     relations.update([r['email'] for r in profile.content.get('relations', [])])
 
-    ## TODO:: Parameterize the number of years for publications to consider from
     ## Publications section: get publications within last n years, default is all publications from previous years
-    for pub in profile.content.get('publications', []):
-        publications.add(pub.id)
+    if n_years:
```

Comment:
I think n_years should be used to filter all the data of the profile that contains a date like: history and relations.

Reply:
Ah, just like neurips. I think I can simply copy it over.

Reply:
OK, @melisabok  I added it to relations and history

Reply:
looks good!

"default" policy checks the whole profile content from the last N years. 
"neurips" policy checks the profile content ignoring intern positions, only including co-worker and co-auhtors relations and use email domains only when there is history record from the last N years.

Reply:
should we rename these policies? what do you think?

Reply:
Are you asking this because you want a name that is more descriptive of what it does?